explainer.md

```diff
@@ -343,24 +348,19 @@ ARIA attributes, they are never reflected. Similarly, Accessible Properties
 do not reflect equivalent implicit properties of their associated DOM elements,
 either.
 
-Specifically, all of the supported attributes on AccessibleNode return null
-initially. It doesn't matter if the associated DOM element has a computed
-role, or if you give it an explicit ARIA role or ARIA attributes - none of
-these are ever reflected in the AccessibleNode object.
-
 ```js
 button = document.createElement("button");
 button.innerHTML = "Next";
-console.log(button.accessibleNode.role);   // null
-console.log(button.accessibleNode.label);  // null
+console.log(button.accessibleNode.role);   // null, not "button"
```

Comment:
Nevermind, see new patch - is this what you had in mind, just that
whole subsection up by one?